kolibri/core/assets/src/navigation.vue

```diff
@@ -42,59 +21,14 @@
           title: 'Kolibri',
           home_link: '/',
         },
-        userNavDisplay: false,
```

Comment:
No, since there is only ever going to be one navigation bar on screen at a time.